### PR TITLE
Feature/administrate dashboards

### PR DESCRIPTION
**Pull request by lamorrison**
Adds _Administrate_ dashboards for `OrganisationUser`s and `Share`s, and lets you set/update the password for all user types (so you can create organisations from the admin interface now)

Added info to the Readme describing what _Administrate_ is used for

### CODE REVIEW

**Comment by mermop** [comment]
Setting the password from the admin interface seems like a bad idea - is it possible to send an email to the new user to set up their own password?

**Comment by Br3nda** [comment]
Perhaps devise invitable

File: README.md [hunk 1]
```diff
@@ -69,3 +69,30 @@ Start a local server
 ```
 
 Test coverage is reported to `coverage/index.html`
+
+## Gems
+
+### [Administrate](https://github.com/thoughtbot/administrate)
```
Comment (Br3nda) [inline README.md]: Perhaps a note in Gemfile, hinting to look at README for more info

**Comment by Br3nda** [comment]
will make an issue to add invitable

**Pull request merged**